<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>360</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>2000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">   background-color: qlineargradient(spread:pad, x0:1, y1:0, x2:1, y2:0, stop:0 rgb(60, 227, 227), stop:1 rgba(0, 0, 128, 255));</string>
  </property>
  <widget class="QLabel" name="TitleText">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>471</width>
     <height>60</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>60</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>90</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Black</family>
     <pointsize>46</pointsize>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{

color:#fff;

background-color:none;

}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string>Tetris Game</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="LaunchButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>290</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>90</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Tw Cen MT Condensed Extra Bold</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: rgba(255, 255, 255, 0);
border: 0;
color: rgb(150, 150, 150);
}
QPushButton:hover{
background: rgba(255, 255, 255, 45);
color: rgb(255,255,255);
border-radius:10px;
}
</string>
   </property>
   <property name="text">
    <string>Start game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CheckUpdatesButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>290</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>900</width>
     <height>90</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>TF2</family>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: rgba(255, 255, 255, 0);
border: 0;
color: rgb(150, 150, 150);
}
QPushButton:hover{
background: rgba(255, 255, 255, 45);
color: rgb(255,255,255);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>681</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(82, 108, 255)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;
	background-color: rgb(245, 71, 71);
}
QPushButton:hover {
	background-color: rgb(245, 71, 71, 150);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_minimize">
    <property name="geometry">
     <rect>
      <x>608</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;
	background-color: rgb(254, 188, 69);
}
QPushButton:hover {
	background-color: rgb(254, 188, 69, 150);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_maximize">
    <property name="geometry">
     <rect>
      <x>586</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;
	background-color: rgb(50, 196, 50);
}
QPushButton:hover {
	background-color: rgb(50, 196, 50, 150);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>280</y>
     <width>691</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame {
  border-radius:60px ;

    background-color:rgb(255, 255, 255)
}
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="ExitButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>0</y>
      <width>290</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>290</width>
      <height>60</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>30</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background: rgba(255, 255, 255, 0);
border: 0;
color: rgb(150, 150, 150);
font: 30pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
}
QPushButton:hover{
background: rgb(170, 0, 0);
border-radius:10px;
color: rgb(255,255,255);
}
</string>
    </property>
    <property name="text">
     <string>Quit Game</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
